packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -65,12 +73,15 @@
 						</n8n-option>
 					</n8n-select>
 
-					<div v-if="issues.length && !hideIssues" :class="$style.warning">
+					<div
+						v-if="getIssues(credentialTypeDescription.name).length && !hideIssues"
+						:class="$style.warning"
+					>
 						<n8n-tooltip placement="top">
 							<template #content>
 								<TitledList
 									:title="`${$locale.baseText('nodeCredentials.issues')}:`"
-									:items="issues"
+									:items="getIssues(credentialTypeDescription.name)"
```

Comment:
Should we create a computed property for the issues and use that instead? I do agree that we should give up on using `:set`, since I couldn't even find any documentation about it

Reply:
I think function is used here so we make sure it's always triggered. Not 100% sure though, but was worried about introducing a new edge-case if we resort to `computed`

Reply:
Oh right, makes sense. The issues might change even tho the dependencies don't